README.md

```diff
@@ -30,21 +30,23 @@ rules_detekt_sha256 = "see-github-releases-page"
 
 http_archive(
     name = "rules_detekt",
-    urls = ["https://github.com/buildfoundation/bazel_rules_detekt/archive/%s.zip" % rules_detekt_version],
-    strip_prefix = "bazel_rules_detekt-%s" % rules_detekt_version,
     sha256 = rules_detekt_sha256,
+    strip_prefix = "bazel_rules_detekt-%s" % rules_detekt_version,
+    url = "https://github.com/buildfoundation/bazel_rules_detekt/archive/%s.zip" % rules_detekt_version,
```

Comment:
If you're so into `tar.gz`s do you want to change that too?

Reply:
I’ll go over it again. Rules packaging is a huge mess.